spec/support/helpers/swagger_example_parser.rb

```diff
@@ -42,7 +42,11 @@ def extract_swagger_properties(properties:, version:)
           extract_swagger_properties(properties: value[:properties], version:)
         # If the value is an array of items, we need to extract the example from the items properties.
         elsif value[:type] == :array || value.key?(:items)
-          [extract_swagger_properties(properties: value[:items][:properties], version:)]
+          if value[:items].key?(:enum)
```

Comment:
had to make this changes as [npq courses](https://github.com/DFE-Digital/npq-registration/blob/main/spec/swagger_schemas/models/participant.rb#L39) has `type: :array`with no `properties` in `items:` as it’s an array of strings (not objects).